api/k8s/v1/k8s.proto

```diff
@@ -166,3 +174,35 @@ message ResizeHPARequest {
 
 message ResizeHPAResponse {
 }
+
+message Deployment {
+  option (clutch.api.v1.id).patterns = {
+    type_url : "clutch.k8s.v1.Deployment",
+    pattern : "{cluster}/{namespace}/{name}"
+  };
+
+  string cluster = 1;
+  string namespace = 2;
+  string name = 3;
+
+  map<string, string> labels = 4;
+  map<string, string> annotations = 5;
+}
+
+message UpdateDeploymentRequest {
+  option (clutch.api.v1.id).patterns = {
+    type_url : "clutch.k8s.v1.Deployment",
+    pattern : "{cluster}/{namespace}/{name}"
+  };
+
+  string clientset = 1 [ (validate.rules).string = {min_bytes : 1} ];
+  string cluster = 2 [ (validate.rules).string = {min_bytes : 1} ];
+  string namespace = 3 [ (validate.rules).string = {min_bytes : 1} ];
+  string name = 4 [ (validate.rules).string = {min_bytes : 1} ];
+
+  map<string, string> labels = 5 [ (validate.rules).map.keys.string = {min_bytes : 1} ];
+  map<string, string> annotations = 6 [ (validate.rules).map.keys.string = {min_bytes : 1} ];
```

Comment:
what are the update semantics?

let's say existing labels are 
```
foo: bar
baz: yes
```

and i post 
```
bop: bop
```

is it 
```
```
bop: bop
```

or 

```
foo: bar
baz: yes
bop: bop
```

Reply:
Its a merge of all labels, so in your example it would be 
```
foo: bar
baz: yes
bop: bop
```